src/Utils/LocalizeVector.js

```diff
@@ -0,0 +1,14 @@
+const from = undefined,
+      skipUpdate = true;
+
+var globalPoint = new PIXI.Point();
+
+function localizeVector(localDisplayObject, globalVector, localVector) {
+    localDisplayObject.getGlobalPosition(globalPoint);
+    globalPoint.x += globalVector.x;
+    globalPoint.y += globalVector.y;
```

Comment:
Instead of creating a new Point each time to store the global position, I'm using always the same instance.